INN/umbrella-caribbean
# Styling for our journalists section

## Changes

This pull request makes the following changes:

- Adds styling for the Our Journalists section

![screencapture-caribbean-test-2019-12-04-12_39_38](https://user-images.githubusercontent.com/18353636/70166735-575df380-1693-11ea-9f27-1bbdda405912.png)

## Why



For #9

## Testing/Questions

Features that this PR affects:

- Our Journalists section


Questions that need to be answered before merging:


Steps to test this PR:

1. Add a text widget to the Homepage Right widget area
2. Paste this text in the text widget: (note that we'll need to replace `http://cijn.wpengine.com` with `https://cijn.org` when pasting on the live site
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
<ul class="our-journalists">
 	<li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image2-1-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/carla-bridglal/">CARLA BRIDGLAL</a></li>
 	<li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image5-2-e1575405486213-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/neil-marks/">NEIL MARKS</a></li>
 	<li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image1-1-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/ivan-cairo/">IVAN CAIRO</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image3-1-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/steffon-campbell/">STEFFON CAMPBELL</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image2-e1575404572823-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/nazima-raghubir/">NAZIMA RAGHUBIR</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image5-1-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/alva-solomon/">ALVA SOLOMON</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image3-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/mark-bassant/">MARK BASSANT</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image1-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/aurora-hererra/">AURORA HERERRA</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image2-e1575421668474-96x96.png" /> <a class="journalist-name" href="//cijn.org/author/dale-elliott/">DALE ELLIOTT</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image5-3.jpg" /> <a class="journalist-name" href="//cijn.org/author/anika-kentish">ANIKA KENTISH</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image3-2-96x96.jpg" /> <a class="journalist-name" href="//cijn.org/author/ingrid-arnesen/">INGRID ARNESEN</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/george-davis.jpg" /> <a class="journalist-name" href="//cijn.org/author/george-davis">GEORGE DAVIS</a></li>
    <li><img src="http://cijn.wpengine.com/wp-content/uploads/2019/12/image5.jpg" /> <a class="journalist-name" href="//cijn.org/author/simon-cottee">SIMON COTTEE</a></li>
</ul>
```
3. Verify it looks ok on desktop and mobile
